static/js/pm_list.js

```diff
@@ -1,52 +1,99 @@
 import $ from "jquery";
+import _ from "lodash";
+
+import render_more_private_conversations from "../templates/more_pms.hbs";
 
-import * as narrow_state from "./narrow_state";
-import * as people from "./people";
 import * as pm_list_data from "./pm_list_data";
 import * as pm_list_dom from "./pm_list_dom";
+import * as resize from "./resize";
 import * as stream_popover from "./stream_popover";
 import * as ui from "./ui";
 import * as ui_util from "./ui_util";
 import * as vdom from "./vdom";
 
 let prior_dom;
-let private_messages_open = false;
 
 // This module manages the "Private messages" section in the upper
 // left corner of the app.  This was split out from stream_list.js.
 
+// To keep a track of the state of private messages section.
+let private_messages_collapsed = false;
+
+// To keep a track whether we're zoomed In or not.
+let zoomed = false;
+
+export function show_more_private_conversations() {
+    zoomed = true;
+}
+
+export function hide_more_private_conversations() {
+    zoomed = false;
+}
```

Comment:
These two functions don't have to be exported.

Reply:
Actually, I think it would be easier to read the code if you didn't make a function for these! When I saw `hide_more_private_conversations` somewhere else, I thought it did a lot more functionality, but it actually just changes this variable! I think it's nice to show that instead of calling the function.

Reply:
Hmm, Agreed!
Updated the PR removing this function and directly used `zoomed = true/false` wherever needed.